detector/calorimeter/ECalBarrel_NobleLiquid_InclinedTrapezoids_o1_v01_geo.cpp

```diff
@@ -560,7 +560,7 @@ static dd4hep::detail::Ref_t createECalBarrelInclined(dd4hep::Detector& aLcdd,
   envelopePhysVol.addPhysVolID("system", xmlDetElem.id());
   caloDetElem.setPlacement(envelopePhysVol);
 
-  // Create caloData object
+ /* // Create caloData object
```

Comment:
I would suggest unstaging this file from the PR. Please, see below the option of adding a few lines to the plugin so that it can remove any assigned data extension